jwst/cube_build/ifu_cube.py

```diff
@@ -20,6 +20,7 @@
 from . import coord
 
 log = logging.getLogger(__name__)
+log.addHandler(logging.StreamHandler())
```

Comment:
Just curious as to what ``StreamHandler`` is used for?

Reply:
I had to add StreamHandler to the code to get my debug statements to print the screen. Jonathan suggested this when I did not get any of my debug statements to print out. I do hope I understood him correctly. I now get the debug statements to print out, but they always print out even when I do not have debug set up by using  --debug.
So how do you (@hbushouse) control printing debug statements in code ?

Reply:
Huh, I've never encountered that problem. Usually just having line 24 (`log.setLevel(logging.DEBUG)`) in conjunction with specifying "-v" on the command line results in all the debug messages appearing. Or are you talking about from within a python session (as opposed to command line)?

Reply:
I took out the StreamHandler and now the debug works for me. I don't know why it was not before.